octoprint_psucontrol/__init__.py

```diff
@@ -508,13 +509,16 @@ def turn_psu_on(self):
             time.sleep(0.1 + self.config['postOnDelay'])
 
             self.check_psu_state()
+            
+            threading.Timer(self.config['postConnectDelay'], self.connect_printer).start()

```

Comment:
What's the reasoning or thought behind using a non-blocking timer?

Reply:
With a blocking timer or sleep a part of the ui was freezing. And I also couldn't abort the timer.